python/ThirteenTeV/Higgs/HH/ResonanceDecayFilter_example_HHTo2B2V_2LIncl_madgraph_pythia8_CP5_cff.py

```diff
@@ -0,0 +1,56 @@
+import FWCore.ParameterSet.Config as cms
+
+# Link to cards:
+# https://github.com/cms-sw/genproductions/tree/aa244debe3894ae9c80c846342db3366f7cacb45/bin/MadGraph5_aMCatNLO/cards/production/13TeV/NonRes_hh/tatabb_node_sm
+
+import FWCore.ParameterSet.Config as cms
+from Configuration.Generator.Pythia8CommonSettings_cfi import *
+from Configuration.Generator.MCTunes2017.PythiaCP5Settings_cfi import *
+
+generator = cms.EDFilter("Pythia8HadronizerFilter",
+    maxEventsToPrint = cms.untracked.int32(1),
+    pythiaPylistVerbosity = cms.untracked.int32(1),
+    filterEfficiency = cms.untracked.double(1.0),
+    pythiaHepMCVerbosity = cms.untracked.bool(False),
+    comEnergy = cms.double(13000.),
+    PythiaParameters = cms.PSet(
+        pythia8CommonSettingsBlock,
+        pythia8CUEP8M1SettingsBlock,
```

Comment:
Hi @covarell , you should change this line to `pythia8CP5SettingsBlock` to use properly the CP5Tune